README.md

```diff
@@ -2,17 +2,68 @@
 
 ## Objectives
 
-1. Practice defining stateless functions as React components
-2. Practice using stateless functions as React components
+1. Practice writing a presentation component with minimal UI state.
+2. Practice writing a presentation component as a stateless function
 
 ## Overview
 
-This lab should be pretty easy. We want to test that students know how to write
-their own stateless functions as components (we should be able to test that they
-spit out markup without backing instances) and that they can use them (we can
-write a few components and ask students to include them in small widget-y
-things).
+Presentation component are also frequently called "dumb" components. Why? Because
+other than rendering themselves, they really don't know how to do much else. But,
+as we've seen, there's something blissful about being "dumb" if you are a component.
 
-## Resources
+![Dumb and Dumber](https://media.giphy.com/media/2dcW1Dlu2sZnW/giphy.gif)
+
+In this lab, we'll illustrate this principle by building two components:
+a `DumbComponent` and a `DumberComponent`. Our `DumbComponent`, will be presentational,
+but it will have a tiny bit of state and therefore it will be less stable than
+our `DumberComponent` -- specifically our `DumbComponent` will experience wild
+mood fluctuations. Our `DumberComponent`, meanwhile, will be written as a
+"stateless functional" component and therefore it will be steadfastly happy.
+
+## DumbComponent
+
+The specs for our `DumbComponent` are as follows:
+
+1. In the `components/DumbComponent.js` file, create a `DumbComponent` component.
+2. The component should be declared as a class (instance of `Component`) so
+   that it can carry state.
+3. The component should have a state property called `mood` that has a default
+   value `happy`.
+4. The component should simply render its current `mood` state to the page in
+   a div.
+5. The component should implement a `handleClick` function that can serve as a
+   callback to the <div>'s click event. When clicked the component's mood 
+   should toggle between happy and sad states.
+
+The fact that this components mood fluctuates when clicked,
```

Comment:
This **component's**? Just a suggestion, possession is always kind of finicky (as a non-native speaker, anyway)